build.sbt

```diff
@@ -105,6 +105,14 @@ lazy val infra = project
         s"-Dscaladex.elasticsearch.port=$elasticsearchPort"
       )
     },
+    reStart / javaOptions ++= {
```

Comment:
Do we still need `reStart`? Why not use `bgRun` instead?

I wonder why `reStart / javaOptions` does not contain `Compile / run / javaOptions` by default. I think this should work:
```scala
reStart / javaOptions := (Compile / run / javaOptions).value
```